server.js

```diff
@@ -71,11 +71,16 @@ function StaticServer(options) {
   this.port = options.port;
   this.rootPath = path.resolve(options.rootPath);
   this.followSymlink = !!options.followSymlink;
-  this.index = options.index || DEFAULT_INDEX;
   this.templates = {
+    'index': (options.templates.index || DEFAULT_INDEX),
     'notFound': options.templates.notFound
   };
 
+  if (options.index) {
+    console.log("options.index is now deprecated please use options.templates.index instead.");
```

Comment:
This should be removed after the next version, should have given people time to update their code.